app/commands.py

```diff
@@ -339,15 +339,8 @@ async def recent(ctx: Context) -> str | None:
         rank = score.rank if score.status == SubmissionStatus.BEST else "NA"
         l.append(f"PASS {{{score.pp:.2f}pp #{rank}}}")
     else:
-        # XXX: prior to v3.2.0, bancho.py didn't parse total_length from
-        # the osu!api, and thus this can do some zerodivision moments.
-        # this can probably be removed in the future, or better yet
-        # replaced with a better system to fix the maps.
-        if score.bmap.total_length != 0:
-            completion = score.time_elapsed / (score.bmap.total_length * 1000)
-            l.append(f"FAIL {{{completion * 100:.2f}% complete}})")
-        else:
-            l.append("FAIL")
```

Comment:
migrating away from this would require a script to fix people's `total_length` in the case they're 0

Reply:
This fix is also *really" old, I don't think any maps with 0 length still exist